### ISSUE TITLE
com.android.build.api.transform.TransformException

### ISSUE THREAD

**Issue by PGMacDesign** [PGMacDesign/PGMacTips#3]
Receiving this issue when compiling this into a different project:

```
Note: Recompile with -Xlint:unchecked for details.
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:transformClassesWithMultidexlistForDebug'.
> com.android.build.api.transform.TransformException: Error while generating the main dex list.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 49s
```

Seems to be related to having this line in the build.gradle file:

`api 'com.google.android.gms:play-services-maps:15.0.1'`

Will look into it further

**Comment by PGMacDesign** [comment]
It seems that this is related instead to someone using this library with a different version of Realm as it conflicts with the DatabaseUtilities class and a few others. Will work on a fix soon

**Comment by PGMacDesign** [comment]
Issue resolved in [Build 0.0.66](https://github.com/PGMacDesign/PGMacTips/releases/tag/0.0.66)

Core problem was related to [this issue](https://github.com/realm/realm-java/issues/1721) in Realm and was solved by following the logic displayed [here](https://realm.io/docs/java/latest/#schemas). 

The solution was to declare a custom Realm Module 

```java
@RealmModule(library = true, allClasses = true)
    public static class PGMacTipsModule {}
```

If you are using a version below 0.0.66, you can declare your own custom module (exclude the library = true param) and rename it whatever you want. Then, whenever you build your RealmConfiguration, use your custom module there:

```java
 builder.modules(new YourCustomModuleHere());
```
